Dockerfile

```diff
@@ -1,23 +1,17 @@
-FROM openmicroscopy/omero-ssh-c7:0.1.0
+FROM openmicroscopy/omero-ssh
```

Comment:
sorry I missed the comment not really sure `omero-ssh-cmd-c7-docker` ?

I am guessing `omero-ssh-c7-docker` should be really `omero-ssh-cmd-c7-docker`
then nonsystemd would be `omero-ssh-c7-docker` but probably is too late to change them